### PR TITLE
[WIP] ODS-354 test added

### PR DESCRIPTION
**Pull request by heyselbi**
Robot framework test for ODS-354: Verify logged in users is displayed in the dashboard

### CODE REVIEW

File: tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Go to Admin View of All Users
+  [Tags]  Sanity    ODS-354
+  Click Link    Admin
+  Wait Until Page Contains Element  id=add-users
+
+Verify User and Admin Are There
+  [Tags]  Sanity    ODS-354
+  Page Should Contain   ${OCP_ADMIN_USER.USERNAME}
+  Page Should Contain   ${TEST_USER.USERNAME}
+  Capture Page Screenshot
```
Comment (crobby) [inline tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot:37]: Newline

**Comment by vasukulkarni** [comment]
Based on discussion with Landon, making it as WIP

File: tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot [hunk 2]
```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Resource         ../../../Resources/ODS.robot
+Resource         ../../../Resources/Common.robot
+Resource         ../../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Suite Setup      Begin Web Test
+Suite Teardown   End Web Test
+
+*** Test Cases ***
+Logout and Log Back in as an Admin
```
Comment (jgarciao) [inline tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot:9]: Title should be the same or similar as in Polarion: `Verify "logged in users" are displayed in the dashboard`

File: tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  Click Link  id:logout
+  Wait Until Page Contains  Successfully logged out.
+  Click Link    id:login
+  Wait Until Page Contains  Log in with  10
+  Login To Jupyterhub  ${OCP_ADMIN_USER.USERNAME}  ${OCP_ADMIN_USER.PASSWORD}  ${OCP_ADMIN_USER.AUTH_TYPE}
+  ${authorization_required} =  Is Service Account Authorization Required
+  Run Keyword If  ${authorization_required}  Authorize jupyterhub service account
+  Wait Until Page Contains Element  xpath://span[@id='jupyterhub-logo']
+
+Go to Admin View of All Users
```
Comment (jgarciao) [inline tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot:21]: This should be a keyword called from the main test (or just add this commands into the main test)

File: tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  [Tags]  Sanity    ODS-354
+  Click Link    Admin
+  Wait Until Page Contains Element  id:add-users
+
+Verify User and Admin Are There
```
Comment (jgarciao) [inline tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot:26]: This should be a keyword called from the main test (or just add this commands into the main test)

File: tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+Resource         ../../../Resources/ODS.robot
+Resource         ../../../Resources/Common.robot
+Resource         ../../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Suite Setup      Begin Web Test
+Suite Teardown   End Web Test
+
+*** Test Cases ***
+Verify "logged in users" are displayed in the dashboard
+  [Tags]  Sanity    ODS-354
+  Launch JupyterHub From RHODS Dashboard Dropdown
```
Comment (bdattoma) [inline tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot:11]: I think it should be replaced by "Can Launch Jupyterhub" to address the change in Dashboard links to launch applications.

**Comment by aloganat** [comment]
Can one of the admins verify this patch?

**Comment by jgarciao** [comment]
Hi  @heyselbi  We are working on this test in #199 Do you mind if we close this PR?

**Comment by heyselbi** [comment]
No problem at all, sounds good! @jgarciao

**Comment by jgarciao** [comment]
Thanks!

**Pull request closed without merging**